ubyssey/templates/headers/topbar.html

```diff
@@ -25,14 +25,19 @@
                 <li {% if section == 'science' %}class="current"{% endif %}>
                     <a href="{% url 'page' 'science' %}">Science</a>
                 </li>
-                <li {% if section == 'events' %}class="current"{% endif %}>
-                    <a href="{% url 'events' %}">Events</a>
-                </li>
-                <li {% if section == 'podcast' %}class="current"{% endif %}>
-                    <a href="{% url 'podcasts' 'episodes' %}">Podcast</a>
-                </li>
-                <li {% if section == 'magazine' %}class="current"{% endif %}>
-                    <a href="{% url 'magazine-landing' %}">Magazine<span class='upper-right-decorator'>New</span></a>
+                <li {% if section == 'events' or section == 'podcast' or section == 'magazine' %}class="current"{% endif %} id="sections-more-dropdown">
```

Comment:
Since 'More section" might grow in the future, we can even do the if-checks using an array
```
{% if section in ['events', 'podcast', 'magazine'] %}
```

Reply:
Yea ill fix this up